jgrapht-ext/src/test/java/org/jgrapht/ext/DOTExporterTest.java

```diff
@@ -43,8 +43,10 @@
 
     // TODO jvs 23-Dec-2006: externalized diff-based testing framework
 
-    private static final String UNDIRECTED = "graph G {" + NL + "  1 [ label=\"a\" ];" + NL
-        + "  2 [ x=\"y\" ];" + NL + "  3;" + NL + "  1 -- 2;" + NL + "  3 -- 1;" + NL + "}" + NL;
+    private static final String UNDIRECTED = DOTExporter.DONT_ALLOW_MULTIPLE_EDGES_KEYWORD
```

Comment:
A general principle, you never depend the tests on the source code. 

Leave the tests as they are. This is the point of the tests, to capture changes in the source code that break things. 

If I break the exporter by changing `DOTExporter.UNDIRECTED_GRAPH_KEYWORD` to `foobar` the test will still work!